src/internal/components/VisuallyHidden.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components'
```

Comment:
Non-blocking nitpick: instead of putting "internal" components in the "internal" directory, I've been using the convention of prepending a `_` to the private code's file name.

I'm happy with either direction, but I think we should be consistent about it.

Could we do a follow-up to handle "internal" component files consistently?

Reply:
Definitely! For background, the `src/internal` bit comes from the internal modules ADR: https://github.com/primer/react/blob/main/contributor-docs/adrs/adr-016-internal-modules.md to help prevent things from being imported unexpectedly. 

I made https://github.com/primer/react/issues/3230 to try and capture what we could look into moving over, let me know if I missed anything! 👀

Reply:
Ahh - I totally forgot about that ADR. Thanks!